cclib/parser/turbomoleparser.py

```diff
@@ -963,10 +1001,10 @@ def extract(self, inputfile, line):
         ## For UHF:
         #       occ. orbital   energy / eV   virt. orbital     energy / eV   |coeff.|^2*100
         #         7 a   alpha          -15.12     12 a   alpha            4.74       24.5
-        #         7 a   beta           -15.12     12 a   beta             4.74       24.5        
-        if "Excitation energy:" in line and "excitation" in self.last_lines[-5].split():
+        #         7 a   beta           -15.12     12 a   beta             4.74       24.5
+        if "Excitation energy:" in line and "excitation" in self.inputfile.last_lines[-6].split():

```

Comment:
For the index change from -5 to -6, what's the trigger line look like?  It should probably go as a code comment.

Reply:
This is a bit weird but the trigger's not actually changed, it's just that the new `last_lines` includes the current line automatically, while before it was only updated at the end of the log file parsing for the current line.

The trigger section is in the comment block above, looks like this:
```
        # Excited state info from escf.
        #                          1 singlet a excitation
        # 
        # 
        #  Total energy:                           -112.9060086800507    
        # 
        #  Excitation energy:                      0.3111453714493050    
```

Reply:
Right, my brain went right to thinking it was `line.split()[-6]`.  This makes sense.